WISE-Community/WISE-Client
# feat(Add Lesson): Combine Create Your Own and Lesson Structure choices

## Notes
- @breity please style as you see fit

## Changes
- Combine Add New Lesson and Add Structure buttons into one and make UX consistent with Add Step flow ([Mockup](https://balsamiq.cloud/s8brvqj/p9j1not/r11F9))
- Move 
   - ```/unit/x/structure/[jigsaw,self-directed-investigation,...]``` to 
   - ```/unit/x/add-lesson/structure/[jigsaw,self-directed-investigation,...]``` 
- Remove "Back to Unit List" button from project authoring view

## Test
- Add Lesson view
   - Create Your Own path works as before, including back/cancel buttons
   - Each of the structure authoring paths works as before, including back/cancel buttons
- Project authoring view no longer shows Back to Unit List button  

Closes #1467
